src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,633 @@
+# -*- coding: utf-8 -*-
+
+"""Download and convert IntAct to BEL.
+
+Run with ``python -m bio2bel.sources.intact``
+
+IntAct is a interaction database with information about interacting proteins, their relation, and the experiments
+in which these interactions were found. Among the interactions that are documented in IntAct are protein modifications,
+associations, direct interactions, binding interactions and cleavage reactions.
+These interactions were grouped according to their biological interpretation and mapped to the corresponding BEL relation.
+The interactions in IntAct had a higher granularity than the interactions in BioGRID. Especially with respect to the
+protein modifications, exact terms specified the relations between the proteins here. Although many protein modifications
+hat corresponding terms in BEL, there were some interaction types in IntAct that could not be mapped directly. Therefore,
```

Comment:
typo.

Explicitly mention that you're talking about mapping to post-translational modificaiton relationshisps. Provide an example BEL statement. Mention explicitly that there's a default BEL namespace of protein modifications. Link to reference material - BEL documentation, PyBEL source file, even via intersphinx with 
```
:data:`pybel.language.pmod_namespace`
```